drupal_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -x
+set -e
+
+REPO=`pwd`
+cd /tmp
+rm -rf drupal
```

Comment:
should we name the folder `drupal-twig-test` or something like that, to be less likely to remove some meaningful drupal tmp content when running this locally ?

Reply:
Ok